### PR TITLE
M3cleanup

### PR DESCRIPTION
**Pull request by mosabua**
Some minor relabellings related to the maven integration and link to env-vars.html from inline help where I expected it to be..

### CODE REVIEW

File: hudson-maven-legacy/maven-plugin/src/main/resources/hudson/maven/MavenModuleSet/global.jelly [hunk 1]
```diff
@@ -26,7 +26,7 @@ THE SOFTWARE.
   Config page
 -->
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-  <f:section title="${%Maven Project Configuration}">
+  <f:section title="${%Maven 2/3 Legacy Project Configuration}">
```
Comment (mcculls) [inline hudson-maven-legacy/maven-plugin/src/main/resources/hudson/maven/MavenModuleSet/global.jelly:29]: Note that this changes the message key used in "hudson-maven-legacy/maven-plugin/src/main/resources/hudson/maven/MavenModuleSet/global*.properties", so these translation entries will also need updating to match the new key.

Comment (mosabua) [inline hudson-maven-legacy/maven-plugin/src/main/resources/hudson/maven/MavenModuleSet/global.jelly:29]: Ok..I will do that later tonight

Comment (mcculls) [inline hudson-maven-legacy/maven-plugin/src/main/resources/hudson/maven/MavenModuleSet/global.jelly:29]: no worries, I already fixed it on master :)

## 

Cheers, Stuart

Comment (mosabua) [inline hudson-maven-legacy/maven-plugin/src/main/resources/hudson/maven/MavenModuleSet/global.jelly:29]: Thanks for covering my bad. I hope there weren't any others. - I checked. The rest was fine.

**Pull request merged**